tests/python/unittest/test_loss.py

```diff
@@ -206,8 +206,9 @@ def test_ctc_loss():
     l = loss(mx.nd.ones((2,25,4)), mx.nd.array([[2,1,3,3],[3,2,2,3]]), mx.nd.array([20,20]), mx.nd.array([2,3]))
     mx.test_utils.assert_almost_equal(l.asnumpy(), np.array([18.82820702, 16.50581741]))
 
-
-@with_seed(1234)
+# @vrakesh: Get rid of fixed seed, as flakiness cannot be reporduced
+# Issue tracked at https://github.com/apache/incubator-mxnet/issues/11694
```

Comment:
Feel free to remove those comments and simply link the issue in PR description.

Reply:
done